fail build with ffmpeg master

### mpv version and platform versions

7314253180e7c5d06bec6d1fed79ce92bff4b6be in archlinux
3e24e8108d62eeee5453aee206370a273ab12a24 ffmpeg

### Reproduction steps

build mpv by using mpv-build script with ffmpeg master branch

### Expected behavior

build ok

### Actual behavior

fail build

### Log file

https://0x0.st/-t9V.txt
~~~
[ 59/529] Compiling player/sub.c
../demux/demux_lavf.c: In function 'update_read_stats':
../demux/demux_lavf.c:273:31: error: 'AVIOContext' has no member named 'bytes_read'
  273 |         int64_t cur = nest->id->bytes_read;
      |                               ^~

Waf: Leaving directory `/tmp/makepkg/mpv-build-git/src/mpv-build/mpv/build'
Build failed
 -> task in 'objects' failed with exit status 1 (run with -v to display more information)
~~~

greetings

--- a/wscript
+++ b/wscript
@@ -415,7 +415,7 @@ FFmpeg libraries. Git master is recommended."
         'name': '--ffmpeg-strict-abi',
         'desc': 'Disable all known FFmpeg ABI violations',
         'func': check_true,
-        'default': 'disable',
+        'default': 'enable',
     }
 ]
 

